### ISSUE TITLE
FHIR -> CDA: missing template-id

### ISSUE THREAD

**Issue by ziegm** [ahdis/cda-fhir-maps#24]
Transformation of the [FHIR-Document](https://github.com/hl7ch/cda-fhir-maps/blob/master/input/ch-emed/card.json)

`<templateId root="1.3.6.1.4.1.19376.1.5.3.1.4.7.1"/>` is missing in the transformed cda document for the marcoumar section

**Comment by oliveregger** [comment]
the medication card example needs to add the dosagetype

**Comment by oliveregger** [comment]
maybe we should add a description of the different dosage types to the ig

**Comment by oliveregger** [comment]
actually the card.json was not conformant to the 'http://fhir.ch/ig/ch-emed/StructureDefinition/ch-emed-dosage-structured', that's why the normal dosing was not added.

**Comment by oliveregger** [comment]
and marcoumar has no dosage, so "normal dosage" template does not apply

**Comment by ziegm** [comment]
### How do we represent non-structured dosing in CDA?

[IHE Suppl PRE](https://www.ihe.net/uploadedFiles/Documents/Pharmacy/IHE_Pharmacy_Suppl_PRE.pdf):
* 6.3.4.6 Dosage Instructions Content Module (1.3.6.1.4.1.19376.1.9.1.3.6) (page 44)
   * Dosage Instructions may be provided structured and/or narrative.
   * If dosage instructions are provided narrative only, the Dosage Instructions data elements SHALL NOT be present as a whole.
* 6.3.4.6.3.3 Dosage Instructions Additional Template ID (page 46)
   *  In case (structured) dosage instructions are not provided (e.g., because they are only provided as narrative text), this element SHALL be set to “Normal Dosing”.

@oliveregger 
* So wie ich es verstehe, sollten wir die unstrukturierte Dosierung mit der templateId für Normal Dosing ergänzen. 
* Und der Text der unstrukturierten Dosierung müsste templateId=2.16.756.5.30.1.1.10.4.52 anstelle templateId=2.16.756.5.30.1.1.10.4.37 sein oder? ([Dosage Instructions Non Structured Entry Content Module](https://art-decor.org/art-decor/decor-templates--cdachemed-?section=templates&id=2.16.756.5.30.1.1.10.4.52&effectiveDate=2016-09-13T15:33:18&language=en-US))

```xml
              <entryRelationship typeCode="COMP">
                <substanceAdministration classCode="SBADM" moodCode="INT">
                  <!--<templateId root="2.16.756.5.30.1.1.10.4.37"/>-->
                  <templateId root="2.16.756.5.30.1.1.10.4.52"/>
                  <text>Take one every two weeks (unstructured dosage)
                    <reference value="#refpdf"/>
                  </text>
                  <consumable>
                    <manufacturedProduct>
                      <manufacturedMaterial nullFlavor="NA"/>
                    </manufacturedProduct>
                  </consumable>
                </substanceAdministration>
              </entryRelationship>
```

**Comment by oliveregger** [comment]
1. verify the behavior with quentin for normal dosing
2. we have to check with quentin for what 2.16.756.5.30.1.1.10.4.52 /  2.16.756.5.30.1.1.10.4.37 are used and if we should switch to  2.16.756.5.30.1.1.10.4.52 (and if the other is used?)

**Comment by oliveregger** [comment]
@qligier i forgot to mention this for the call today, could you have a quick look? for normal dosing if only unstructured and which templateId to use?

**Comment by qligier** [comment]
> So wie ich es verstehe, sollten wir die unstrukturierte Dosierung mit der templateId für Normal Dosing ergänzen.

That seems right.

For the templateIds, they're not defined by IHE so there's not much description to read about. As I understand it, `2.16.756.5.30.1.1.10.4.52` is for completely unstructured dosage and `2.16.756.5.30.1.1.10.4.37` is a reference to the dosage intake mode (so the narrative description of `routeCode` and `approachSiteCode` I guess). We only used the former in structured dosage, but I can't guarantee it's right.

So I'd go with `2.16.756.5.30.1.1.10.4.52` in this case. I've opened a [ticket](https://art-decor.org/art-decor/decor-issues--ch-pharm-?id=2.16.756.5.30.1.127.77.4.6.44) to improve description of these templates.

_Edit: we only used the former (.52), not the latter._
